tensorboard/components/tf_ng_tensorboard/core/core.effects.ts

```diff
@@ -36,22 +47,27 @@ export class CoreEffects {
   /** @export */
   readonly loadPluginsListing$ = createEffect(() =>
     this.actions$.pipe(
-      ofType(coreLoaded),
-      flatMap(() =>
-        this.coreService
-          .fetchPluginsListing()
-          .pipe(
-            map(
-              (plugins) => pluginsListingLoaded({plugins}),
-              catchError(() => of(pluginsListingFailed()))
-            )
-          )
-      )
+      ofType(coreLoaded, reload),
```

Comment:
Here, I am simply trying to mimic the behavior of tf-tensorboard. Btw, pluginsListing *do* change and environments list technically can change.

Reply:
I see!  Let's be sure to use them in a followup.  Otherwise, if there's no plan to use this code, maybe we can remove for now

Reply:
> Otherwise, if there's no plan to use this code, maybe we can remove for now

What do you mean?

Reply:
Ah, I now see why fetching pluginsListing and environment make sense.
Per offline discussion: 
- if a user stops the running TB process and restarts it with a different logdir, the environment can change 
- since we have the notion of active/inactive plugins, the list needs to be refreshed (although ideally we might not need this notion of active/inactive)